configs/server/forseti_conf_server.yaml.in

```diff
@@ -183,6 +183,14 @@ inventory:
         #    - spanner.googleapis.com/Instance
         #    - sqladmin.googleapis.com/Instance
         #    - storage.googleapis.com/Bucket
+        #    - container.googleapis.com/Cluster
```

Comment:
container.googleapis.com/Cluster is already included above.

Reply:
Done